### ISSUE TITLE
Initialise Project Code Base

### ISSUE THREAD

**Issue by mike-hanson** [Playology/fire-testing#4]
As the Owner
I need the initial codebase setup
So that I can refer to it in documentation

**Comment by jamesanderson9182** [comment]
@testpossessed I was reading your README.md and wasn't sure how to 'contact you directly'. 

This project is exactly what I was looking for! **Is there anything I can do to help?** I've got 26 months experience writing angular applications using TDD. We don't use angularfire in work, but I use it in my personal projects outside work. 

I've been working on one personal angular project for about a year and I've only been able to get ~82% coverage because I gave up trying to unit test the services that hold my AngularFire code. I'm basically using the services to give me a way to mock what they return in my component tests. The services themselves have minimal logic, in fact they call out to unit tested mapper and helper classes for almost all non-angularfire code.

**Comment by mike-hanson** [comment]
Sorry, forgot that my email was private on GitHub and they don't have a PM feature.

I find myself in the same boat repeatedly with AngularFire and have to resort to less than ideal ways to test AF code.  It gets messy having to create stubs and mocks all the time.  I have built up a libary of factory functions and fakes that I use, but having to copy them into every project is also a bit tedious so thought it was time to do something more structured and package it.

Been distracted the last couple of weeks on another project, but getting back to spending a bit of time each week on this.

I have made my email address public now so contact me via the link on my profile and let me know what you would be most comfortable doing then we can discuss further.
